scripts/common.sh

```diff
@@ -21,7 +21,7 @@ resolveCommand () {
   fi
 }
 
-if [[ -z $CONTAINER_ENGINE ]]; then
+if ! [ -v CONTAINER_ENGINE ]; then
```

Comment:
What does `-v` do and how portable is it?

Struggling to search out a definition of `-v`

Reply:
```
-v varname

True if the shell variable varname is set (has been assigned a value).
```
https://www.gnu.org/software/bash/manual/bash.html under 6.4 Bash Conditional Expressions

Not sure when it landed in bash

Reply:
`-v` isn't support by Bash 3, which unfortunately is the default on Mac OS X.

```
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin22)
bash-3.2$ if [[ -v FOO ]];  then echo pass; fi
bash: conditional binary operator expected
bash: syntax error near `FOO'
```

```
GNU bash, version 5.2.15(1)-release (x86_64-apple-darwin22.1.0)

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
bash-5.2$ if [[ -v FOO ]];  then echo pass; fi
pass
bash-5.2$
bash-5.2$
bash-5.2$ unset FOO
bash-5.2$ if [[ -v FOO ]];  then echo pass; fi
```

Reply:
I'll go back to setting `+u` tactically